### PR TITLE
Fix public share issue

### PR DESCRIPTION
**Pull request by lpoli**
This PR contains following changes regarding clit tests:
1. Revoke public authticket should fail to download file
2. Collaborator should not be able to share file.

### CODE REVIEW

File: tests/cli_tests/zboxcli_collaborator_system_test.go [hunk 1]
```diff
@@ -186,8 +186,8 @@ func TestCollaborator(t *testing.T) {
 			"allocation": allocationID,
 			"remotepath": remotepath,
 		})
-		require.Nil(t, err, "Error in sharing the file as collaborator", strings.Join(output, "\n"))
-		require.Len(t, output, 1, "Unexpected number of output lines", strings.Join(output, "\n"))
+		require.NotNil(t, err, "expected error when collaborator shares a file")
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_collaborator_system_test.go:189]: This is still getting an error which is causing test to fail

File: tests/cli_tests/zboxcli_share_file_test.go [hunk 2]
```diff
@@ -130,8 +130,7 @@ func TestShareFile(t *testing.T) {
 			"download file - Unexpected output", strings.Join(output, "\n"))
 	})
 
-	// FIXME is auth ticket with no target wallet and expiration meant to be eternal and cannot be revoked?
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_share_file_test.go:133]: If you're removing the FIXME, I suppose the case is the expected behavior?

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

File: tests/cli_tests/zboxcli_share_file_test.go (hunk 2 above)
Comment (lpoli) [reply tests/cli_tests/zboxcli_share_file_test.go:133]: Revoke on public share is possible now

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/public-share-issue  |
| gosdk  | fix/public-share-issue  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by Kishan-Dhakan** [comment]
Closing this, as the changes in this PR have already been merged to master via #192 and #189.

cc @stewartie4 @lpoli

**Pull request closed without merging**